src/Stratis.Bitcoin/EventBus/EventBase.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stratis.Bitcoin.EventBus
+{
+    /// <summary>
+    /// Basic implementation of <see cref="IEvent"/>.
+    /// </summary>
+    /// <seealso cref="Stratis.Bitcoin.EventBus.IEvent" />
+    public class EventBase : IEvent
```

Comment:
We want people to inherit this, but do we expect them to use it or could it be made abstract?

Reply:
good point, this should be abstract, I'll change this.
Probably if we implement a generic EventBase<T> where T is the content of the event, then we can let that class usable, but I'd discourage using common events type because event type should describe the event itself, so any kind of "base event" should be abstract